cmd/masa-node/init.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"io"
+	"log"
+	"os"
+	"os/user"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+
+	masa "github.com/masa-finance/masa-oracle/pkg"
+)
+
+func init() {
+
+	// Set up masa file path based on current user and config settings
+	usr, err := user.Current()
+	if err != nil {
+		log.Fatal("could not find user.home directory")
+	}
+	// Set default values and use constants for values used elsewhere in the application
+	viper.SetDefault("LOG_LEVEL", "info")
+	viper.SetDefault("LOG_FILEPATH", "masa_oracle_node.log")
+	viper.SetDefault(masa.PrivKeyFile, "masa_oracle_key")
```

Comment:
Currently in dev, we have this same situation but not in a separate file (it's still in main) - but these SetDefault lines are setting viper variables with keys as the values of things like -- whatever the value of masa.PrivKeyFile is - 

The syntax for viper.SetDefault should be a text string key, and then a value.